app/about_us/our_values/page.js

```diff
@@ -0,0 +1,67 @@
+import styles from "./page.module.css";
+
+const OurValues = () => {
+  // TASK - React 1 week 1
+  // Create the "Our Values" section
+  // Use the descriptions provided in /app/about_us/README.md
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.ourValues}>
+        <p>OUR VALUES</p>
```

Comment:
This section title was an `<h2>` before, make sure to be consistent on the same page!

Reply:
Ok noted and revert the change to h2